.github/workflows/integration-tests.yaml

```diff
@@ -0,0 +1,40 @@
+name: tests-integration
+run-name: Tests (integration)
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+
+env:
+  TELEPORT_GET_VERSION: v10.2.0
+  TELEPORT_ENTERPRISE_LICENSE: ${{ secrets.TELEPORT_ENTERPRISE_LICENSE }}
```

Comment:
Doesn't seem used/needed here.

Reply:
@gozer - unless things have changed, the Teleport plugin integration tests actually run full, compiled versions of Teleport to test against (as opposed to the Teleport integration tests, which essentially use Teleport as a library). 

`TELEPORT_ENTERPRISE_LICENSE` will be picked up by the test setup code and written to a file for the Teleport instances under test to find.

Reply:
Also, `TELEPORT_ENTERPRISE_LICENSE` is given an exemption from our _no secrets in public CI_ policy, as it's not really a secret, it's just a convenient way to inject a license into the build. 

(This license is well known as a debug/test license so if found in the wild we know it's not valid.)

Reply:
... although, on second thoughts, it might be an idea to talk with whoever is currently working on license enforcement and make sure that these assumptions are still valid.

Reply:
I can't (yet) see anything that would change here with license enforcement. License enforcement will disable the enterprise features when the license has expired (with a grace period) and should not affect the way the license is provided.